frontend/.eslintrc.json

```diff
@@ -6,8 +6,7 @@
     "extends": [
         "airbnb",
         "react-app",
-        "prettier",
-        "prettier/react"
+        "prettier"
```

Comment:
Why did you remove `prettier/react`? Looks like this is accessing additional prettier config specific to react: https://eslint.org/docs/developer-guide/shareable-configs#sharing-multiple-configs

Reply:
Turns out prettier/react was merged into prettier so this is no longer necessary since 8.0